packages/react-components/src/index.ts

```diff
@@ -11,6 +11,7 @@ export { LineChart } from './components/line-chart';
 export { BarChart } from './components/bar-chart';
 export { ScatterChart } from './components/scatter-chart';
 export { KPI } from './components/kpi/kpi';
+export { Gauge } from './components/gauge/gauge';
```

Comment:
lets move this export to a seaparate PR which should include this change + new doc site page for Gauge